kubernetes/filters.go

```diff
@@ -32,22 +32,22 @@ func FilterAuthorizationPoliciesBySelector(workloadSelector string, authorizatio
 	return filtered
 }
 
-func FilterByHost(host, serviceName, namespace string) bool {
+func FilterByHost(host, hostNamespace, serviceName, svcNamespace string) bool {
```

Comment:
Perhaps this deserves a comment that the "hostNamespace" may be included into the "host" string ?

I see that change is being propagated into the validations logic and in theory it should match but the host defined in a DR shouldn't necessarily match the namespace where the DR is defined.

If tests are passing we are good, just please let's leave a comment in cases that there is any regression.

@hhovsepy I think that if make test is green, we are good here, but as you are also updating the validations, just in case that you may have any other aspect into consideration.

Reply:
> Perhaps this deserves a comment that the "hostNamespace" may be included into the "host" string ?
> 
> I see that change is being propagated into the validations logic and in theory it should match but the host defined in a DR shouldn't necessarily match the namespace where the DR is defined.

Yes. Indeed, it may deserve more explanation than that.
The idea is that if `host` is "simple", like `reviews` rather than the full FQDN, you should pass what is the DR namespace in `hostNamespace` to correctly differentiate if the service and the host are really referencing the same thing.

However, if the `host` is already including the namespace on it, the `hostNamespace` parameter becomes irrelevant, because the very first `if` should never match.

I'll add a doc comment to this function.

Reply:
I added the comment.